client/js/views/Stats.vue

```diff
@@ -0,0 +1,151 @@
+        item-text="name"
+      >
+        <template #selection="{ item }">
+          <h1 class="text-h2" data-cy="selected-season-header">
+            {{ item.name }}
+          </h1>
+        </template>
+      </v-select>
+      <h4 v-if="selectedSeason" class="text-h4">
+        <span data-cy="season-start-date">{{ seasonStartString }}</span>
+        -
+        <span data-cy="season-end-date"> {{ seasonEndString }} </span>
+      </h4>
+    </section>
+    <section class="px-8">
+      <!-- Season Champions -->
+      <div class="mb-10">
+        <h2
+          v-if="
+            selectedSeason &&
+              (selectedSeason.bracketLink ||
+                selectedSeason.firstPlace ||
+                selectedSeason.secondPlace ||
+                selectedSeason.thirdPlace)
+          "
+          class="text-h2 mb-4"
+        >
+          Season Champions
+        </h2>
+        <div class="d-flex justify-space-around flex-wrap">
+          <award-card
+            v-if="selectedSeason && selectedSeason.firstPlace"
+            :username="selectedSeason.firstPlace"
+            :place="1"
+            class="mb-4"
+          />
+          <award-card
+            v-if="selectedSeason && selectedSeason.secondPlace"
+            :username="selectedSeason.secondPlace"
+            :place="2"
+            class="mb-4"
+          />
+          <award-card
+            v-if="selectedSeason && selectedSeason.thirdPlace"
+            :username="selectedSeason.thirdPlace"
+            :place="3"
+            class="mb-4"
+          />
+        </div>
+        <p v-if="selectedSeason && selectedSeason.bracketLink" class="text-body-1">
+          Click
+          <a :href="selectedSeason.bracketLink" data-cy="tournament-bracket-link">
+            here to see the official tournament bracket
+          </a>
+        </p>
+        <p v-if="selectedSeason && selectedSeason.footageLink" class="text-body-1">
+          Click
+          <a :href="selectedSeason.footageLink" data-cy="tournament-footage-link">
+            here to watch the official tournament footage
+          </a>
+          with play-by-play commentary.
+        </p>
+      </div>
+      <!-- Rankings Table -->
+      <h2 class="text-h2 ml-4">
+        Weekly Rankings
+        <stats-scoring-dialog />
+      </h2>
+      <stats-leaderboard :loading="loadingData" :season="selectedSeason" />
+    </section>
+  </div>
+</template>
+
+<script>
+import { io } from '@/plugins/sails.js';
+import AwardCard from '@/components/AwardCard.vue';
+import StatsLeaderboard from '@/components/StatsLeaderboard.vue';
+import StatsScoringDialog from '@/components/StatsScoringDialog.vue';
+
+const dayjs = require('dayjs');
+
+export default {
+  name: 'Stats',
+  components: {
+    AwardCard,
+    StatsLeaderboard,
+    StatsScoringDialog,
+  },
+  data() {
+    return {
+      loadingData: false,
+      selectedSeason: null,
+      seasons: [],
+    };
+  },
+  computed: {
+    seasonStartString() {
```

Comment:
`seasonStartFormatted()`or `startSeasonFormatted()` make a bit more sense imo, I like to name variables in a way that it's easy to read what they're doing at a glance.

Reply:
how's `seasonStartDateString`?